### ISSUE TITLE
Automate build process using GitHub Actions

### ISSUE THREAD

**Issue by BZandi** [openPupil/Open-PupilEXT#43]
Currently PupilEXT need to be build manually for macOS and Windows. With GitHub Actions PupilEXT can be build and tested automatically after pull requests. This has a high priority and needs to be implemented in the near future.

**Comment by maciejskorski** [comment]
I agree. Manual building process is error-prone and limits reproducibility, and ultimately also citations.

**Comment by BZandi** [comment]
We uploaded the compiled releases for windows and macOS in GitHub if you face an error during the manual build process.

Follow this link: https://github.com/openPupil/Open-PupilEXT/releases/tag/v0.1.1-beta
